scripts/build-release.sh

```diff
@@ -7,12 +7,14 @@ printf '\n\nChecking TS definitions..\n\n'
 # babel will produce the real *.d.ts file within each package/build dir.
 tsc --outDir $BUILD_DIR
 
-for d in packages/* ; do
-    printf "\n\nBuilding @cennznet/$d...\n\n"
+for d in $(pwd)/packages/* ; do
+    printf "\n\nBuilding $d...\n\n"
     npx babel -f babel.config.js --ignore='**/*.d.ts' --extensions='.ts' --out-file-extension='.js' --copy-files -d "$d/build" "$d/src"
     # copy in essential files for publishing to npm
     cp LICENSE $d/build/LICENSE
-    cp $d/{README.md,package.json} $d/build
+    ls $d
+    cp $d/{README.md} $d/build
+    cp $d/{package.json} $d/build
```

Comment:
lets take no changes 😅 
```suggestion
    cp $d/README.md $d/build/README.md
    cp $d/package.json $d/build/package.json
```

Reply:
ok :) 
Trying to rebase.. having some trouble..

Reply:
something like `git fetch origin; git rebase origin/develop` should do it